specs/RestartRequested.md

```diff
@@ -98,10 +134,21 @@ s
 ```c#
 namespace Microsoft.Web.WebView2.Core
 {
+    enum CoreWebView2RestartRequestedPriority
+    {
+        BestEffort = 0,
+        Critical = 1,
```

Comment:
The term Critical has a special meaning at least for [Windows Updates](https://learn.microsoft.com/en-us/troubleshoot/windows-client/installing-updates-features-roles/standard-terminology-software-updates) which doesn't apply and would be confusing.

And BestEffort is less clear to me where it would lie in terms of normal priority terms (low, normal, high).

Can we rename (BestEffort, Critical) to (Normal, High) or something along those lines?

Reply:
I agree with Normal and High

Reply:
Updated.